charts/firefly/README.md

```diff
@@ -441,10 +440,28 @@ annotation can be set on the `Ingress`:
 > it assumes the provided `hosts` must match the `tls[0].hosts` and that the secret is either pre-made or
 > provided by cert-manager.
 
-### Tokens via ERC1155 Connector
+### Tokens Connectors
+
+#### ERC1155
+
+By default, the chart comes with the [FireFly Tokens ERC1155 connector](https://github.com/hyperledger/firefly-tokens-erc1155)
+enabled.
+
+The ERC1155 connect requires its [ERC1155 smart contract](https://github.com/hyperledger/firefly-tokens-erc1155/blob/main/solidity/contracts/ERC1155MixedFungible.sol)
+to be deployed via Ethconnect. To do so, you can follow the same process for deploying the [FireFly smart contract](#smart-contract-deployment).
+Once its deployed, be sure to provide the contract address to the chart:
+
+```yaml
+config:
```

Comment:
Great point / question. The ERC1155 contract address really is never used in the config templating helper for the `tokens` plugin: https://github.com/hfuss/firefly-helm-charts/blob/tokens-erc1155/charts/firefly/templates/_helpers.tpl#L239-L247

its purely config for ERC1155 not FireFly to your point and this will conflate `config` as we more connectors.

And furthermore, the only bit of config we allow users to provide for the DataExchange as a supporting example is the API key which we did via `dataexchange.apiKey`. If the DataExchange was external so that `dataexchange.enabled` is `false`, there's `config.dataexchangeAPIKey` to provide the value to FireFly: https://github.com/hyperledger/firefly-helm-charts/blob/main/charts/firefly/templates/dataexchange/secret.yaml#L29-L31

So considering that, I agree. Will change the contract address and topic values to live under `erc1155`.